pkg/utils/imagevector/types.go

```diff
@@ -22,9 +22,10 @@ package imagevector
 // in the seed cluster and act on the shoot cluster. Different versions might be used depending on the
 // seed and the shoot version.
 type ImageSource struct {
-	Name           string  `json:"name" yaml:"name"`
-	RuntimeVersion *string `json:"runtimeVersion,omitempty" yaml:"runtimeVersion,omitempty"`
-	TargetVersion  *string `json:"targetVersion,omitempty" yaml:"targetVersion,omitempty"`
+	Name           string    `json:"name" yaml:"name"`
+	RuntimeVersion *string   `json:"runtimeVersion,omitempty" yaml:"runtimeVersion,omitempty"`
+	TargetVersion  *string   `json:"targetVersion,omitempty" yaml:"targetVersion,omitempty"`
+	Architecture   *[]string `json:"architecture,omitempty" yaml:"architecture,omitempty"`
```

Comment:
@rfranzke Architecture field here is fine or do we need to make it.
```suggestion
	Architecture   []string `json:"architecture,omitempty" yaml:"architecture,omitempty"`
```

Reply:
Yes, slices should not have a pointer.

Reply:
/approve cancel

I missed this, yes, let's remove the pointer.

Reply:
Also, the field name should be plural